src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -377,6 +377,9 @@ define('diagramStore', ['jquery', 'draw.io', 'xwiki-events-bridge'], function($)
 
   var updateFormFields = function(event) {
     forEachOpenedFile(function(file) {
+      // This is a workaround for https://github.com/jgraph/drawio/issues/490
+      // Stop editing for getting the latest content from diagram
+      file.ui.editor.graph.stopEditing(false);
```

Comment:
Shouldn't be `true` used here?

Reply:
@acotiuga the parameter controls whether to cancel or not the changes. In this case we don't want to cancel the changes.